yaksh/python_code_evaluator.py

```diff
@@ -29,6 +29,9 @@ def _check_code(self):
             fname, lineno, func, text = info[-1]
             text = str(test_code).splitlines()[lineno-1]
             err = "{0} {1} in: {2}".format(type.__name__, str(value), text)
+        except (SyntaxError, TypeError, NameError, IndentationError, ValueError) :
```

Comment:
NO, do not do this.  Instead catch the TimeoutException first and raise again, else catch the base Exception class. I had mentioned this earlier.